### DISCUSSION TITLE
No toolbox in horizontal boxplot

### DISCUSSION THREAD

**Discussion by paulinecazalis** [apexcharts/apexcharts.js#3292] [Q&A]
Hello ! 

Is it normal that there is no toolbox when the boxplot is horizontal ?

<img width="719" alt="Capture d’écran 2022-07-29 à 14 41 33" src="https://user-images.githubusercontent.com/67122396/181762930-0212ee48-ca15-4011-aed3-b84e625d98e0.png">

I tried to force the display by setting the show parameter for the toolbar to true, but it doesn't work.

However, if the boxplot is not horizontal it is displayed .
<img width="633" alt="Capture d’écran 2022-07-29 à 14 46 40" src="https://user-images.githubusercontent.com/67122396/181763555-5be143d9-1604-4182-b6ea-732b5d3e9f61.png">

Thank you in advance for your reply !

**Comment by junedchhipa** [comment]
Zooming/Panning is not available for inverted (horizontal) charts

**Reply by sudhir612** [reply]
@junedchhipa Is there any way to force boxplot horizontal, to start xaxis from 0. currently it has its own logic for starting the xaxis from.
